README.md

```diff
@@ -31,13 +31,18 @@ own and can obtain certificates for.
 
 ##### Manual installation
 
-  1. Install Go version 1.10 or higher. Optionally, set
+  1. Install Go version 1.10. Optionally, set
      [$GOPATH](https://github.com/golang/go/wiki/GOPATH) to something (default
      is `~/go`) and/or add `$GOPATH/bin` to `$PATH`.
-  2. `go get -u -mod=vendor github.com/ampproject/amppackager/cmd/amppkg`
+  1. `go get -u github.com/ampproject/amppackager/cmd/amppkg`
```

Comment:
-mod=vendor is not available in Go 1.14: https://github.com/GoogleContainerTools/skaffold/issues/3699

I've tested it today with Go 1.10 and it didn't work for me either (but it did work when I removed -mod=vendor):

michaelrybak$ go get -u -mod=vendor github.com/ampproject/amppackager/cmd/amppkg
flag provided but not defined: -mod
usage: get [-d] [-f] [-fix] [-insecure] [-t] [-u] [-v] [build flags] [packages]
Run 'go help get' for details.

Reply:
@twifkak, do we still need to generate the vendor directory?

Reply:
Oof. They're not making it easy. I guess we should list two command lines: with `-mod=vendor` if 1.10-1.13, and without if 1.14+. I think it's too soon to up our minimum version to 1.14 -- e.g. it's not even in the Ubuntu release from 7 days ago: https://packages.ubuntu.com/focal/golang-any, let alone in the previous LTS version.

We could just do as you've suggested, which AFAIK means that the vendor directory will be ignored on Go 1.10-1.13. I'm not wild about that, for two reasons:
 1. I think it's slow? Involves a lot of `git clone`s.
 2. I'm just not knowledgeable enough in how go modules work. After doing all those `git clone`s, does Go verify against `go.sum`? I would prefer some safety check that users are getting the version of the libs that we tested against.

I could easily be convinced on both fronts.

(To make things more confusing, it seems the official documentation is incorrect. https://github.com/golang/go/wiki/Modules#how-do-i-use-vendoring-with-modules-is-vendoring-going-away and https://golang.org/doc/go1.14#go-command seem to suggest continued support for `-mod=vendor`. But the commit that you link to is unequivocal.)